actors/player/PlayerController.gd

```diff
@@ -45,6 +45,8 @@ func fall(gap_size):
 
 func _on_Die_finished(string):
 	set_dead(true)
+	get_tree().change_scene("res://Demo.tscn")
```

Comment:
This shouldn't be in the player script. Rather in a file like Game.gd or something similar.